MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Media/MediaRouter.js

```diff
@@ -14,12 +16,39 @@ class MediaRouter extends OrchestraRouter
      */
     preinitialize(options) {
         this.routes = {
-            'media/list(/:page)': 'listMedia',
-            'media/new'         : 'newMedia'
+            'media/list(/:page)'   : 'listMedia',
+            'media/new'            : 'newMedia',
+            'folder/list'          : 'listFolders',
```

Comment:
You could split folder routes in an other router and use an abstract router to factorize getBreadCrumb